### PR TITLE
added square images and styles

### PR DESCRIPTION
**Pull request by dstrizzi**

### CODE REVIEW

File: src/components/Grid.js [hunk 1]
```diff
@@ -5,9 +5,28 @@ import { StyledGrid } from "../styles/StyledGrid";
...
+          <Square key={x} type={row} />
+        ))}
+      </StyledGrid>
+      <StyledGrid>
+        {grid[3].map((row, x) => (
+          <Square key={x} type={row} />
+        ))}
+      </StyledGrid>
+    </div>
   );
```
Comment (bmorelli25) [inline src/components/Grid.js]: Why this change?

Comment (dstrizzi) [reply src/components/Grid.js]: so this is what we need to discuss. If you can css it to work the other way the I'm all for it, but this was the easiest way for me to get the grid set up how we needed it

Comment (bmorelli25) [reply src/components/Grid.js]: Yeah, I'll take some time and play around with it. It's kinda hacky to hard-code in the grid rows and I want to try and avoid that this time around :)

Comment (dstrizzi) [reply src/components/Grid.js]: Updated it to removed all the mess

File: src/styles/StyledCard.js [hunk 2]
```diff
@@ -10,4 +10,7 @@ export const StyledCard = styled.div`
   box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
   display: flex;
   flex-direction: column;
+  min-width: 142px;
+  max-width: 142px;
```
Comment (bmorelli25) [inline src/styles/StyledCard.js]: Can we just replace these with `width: 142px`?

File: src/styles/StyledSquare.js [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
+    const type = props.type;
+    if (type === 2) {
+      return `url(${coin})`;
+    } else if (type === 3) {
+      return `url(${tree})`;
+    } else if (type === 4) {
+      return `url(${x})`;
+    }
+  }};
+  background-size: 90%;
```
Comment (bmorelli25) [inline src/styles/StyledSquare.js]: Test this out with 70% and let me know what you think

File: src/styles/StyledSquare.js [hunk 4]
```diff
@@ -1,7 +1,34 @@
...
+  width: 30px;
+  background-image: ${(props) => {
+    const type = props.type;
+    if (type === 2) {
+      return `url(${coin})`;
+    } else if (type === 3) {
+      return `url(${tree})`;
+    } else if (type === 4) {
+      return `url(${x})`;
+    }
```
Comment (bmorelli25) [inline src/styles/StyledSquare.js:20]: We may end up moving this logic into the component if we need to, but it makes sense to leave it here for now.

File: src/styles/StyledCard.js (hunk 2 above)
Comment (dstrizzi) [reply src/styles/StyledCard.js]: Potentially. Kinda gets messy if you resize the screen too small but we can fix that with media rules

File: src/styles/StyledSquare.js (hunk 3 above)
Comment (dstrizzi) [reply src/styles/StyledSquare.js]: Love it!

File: src/components/Square.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
 
-import { StyledSquare } from "../styles/StyledSquare";
+import { StyledSquare, StyledEmpty } from "../styles/StyledSquare";
 
 function Square({ type }) {
-  console.log(type);
-  return <StyledSquare>{type}</StyledSquare>;
+  if (type) {
+    return <StyledSquare type={type}> </StyledSquare>;
+  } else return <StyledEmpty> </StyledEmpty>;
```
Comment (bmorelli25) [inline src/components/Square.js]:
```suggestion
  } else return <StyledEmpty />;
```

File: src/styles/StyledCard.js (hunk 2 above)
Comment (bmorelli25) [reply src/styles/StyledCard.js]:
```suggestion
  width: 142px;
```

File: src/styles/StyledSquare.js (hunk 3 above)
Comment (bmorelli25) [reply src/styles/StyledSquare.js]: Neat.

```suggestion
  background-size: 70%;
```

File: src/components/Grid.js [hunk 6]
```diff
@@ -5,9 +5,11 @@ import { StyledGrid } from "../styles/StyledGrid";
...
 function Grid({ grid }) {
   return (
-    <StyledGrid>
-      {grid.map((row) => row.map((cell, x) => <Square key={x} type={cell} />))}
-    </StyledGrid>
+    <div>
+      <StyledGrid>
+        {grid.map((row) => row.map((cell, x) => <Square key={x} type={cell} />))}
+      </StyledGrid>
+    </div>
```
Comment (bmorelli25) [inline src/components/Grid.js]: These divs aren't needed

```suggestion
    <StyledGrid>
      {grid.map((row) => row.map((cell, x) => <Square key={x} type={cell} />))}
    </StyledGrid>
```

File: src/cards/treasureCards.js [hunk 7]
```diff
@@ -4,9 +4,14 @@
 // 3 = x
 // 4 = tree
 
+const grey = "150, 150, 150";
+const green = "55, 161, 28, 63";
+const orange = "245, 147, 49, 96";
+const purple = "178, 121, 224, 88";
```
Comment (bmorelli25) [inline src/cards/treasureCards.js:10]: brillian

Comment (bmorelli25) [reply src/cards/treasureCards.js:10]: t

**Review by bmorelli25 — APPROVED** [review]
Much wow
